src/Moryx.Runtime.SmokeTest/Moryx.Runtime.SmokeTest.csproj

```diff
@@ -5,6 +5,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Description>Runtime RunMode for executing smoke tests.</Description>
     <CreatePackage>true</CreatePackage>
+    <PackageTags>command line parser;moryx;tests;iiot;iot</PackageTags>
```

Comment:
command line parser should be not used as tag here. This class library is used for smoke tests runtime environment

e.g.: "MORYX;Runtime;Test;SmokeTests"